### DISCUSSION TITLE
A toggle for only showing selected MC versions

### DISCUSSION THREAD

**Discussion by woodiertexas** [modrinth/.github#13] [Feature requests]
### Is your feature request related to a problem? Please describe.

If you select MC 1.16.5 on the mods page, it will show mods for 1.16.5 or higher.

### Describe the solution you'd like

A toggle that only shows mods that only have 1.16.5 as their listed version. This is generally helpful for people wanting to port mods only found on older MC versions to the latest MC versions.

**Comment by triphora** [comment]
What is this supposed to mean? Don't we already have this functionality? https://modrinth.com/mods?v=1.16.5

> it will show mods for 1.16.5 or higher.

No, it shows mods that have a version *on* 1.16.5. Are you looking for something that shows mods that *only* have a version on 1.16.5? I'm not entirely sure whether that's possible with MeiliSearch.

**Comment by woodiertexas** [comment]
I meant shows mods only on 1.16.5 (this was an example because i tried searching 1.8.9 mods on CF and i got mods that have a 1.8.9 version but also have versions for the latest MC versions)

**Comment by triphora** [comment]
I'm not entirely sure how or if we could implement this
